packages/record/src/recorder.ts

```diff
@@ -1,58 +1,106 @@
 import Data from "@lookit/data";
-import lookitS3 from "@lookit/data/dist/lookitS3";
+import {
+  default as lookitS3,
+  default as LookitS3,
+} from "@lookit/data/dist/lookitS3";
```

Comment:
Importing two different defaults from the same file?

Reply:
Or I suppose the same default saved to two different vars?

Reply:
Probably a rebase error.